README_GAMELIFT.md

```diff
@@ -2,39 +2,110 @@
+`
+aws gamelift upload-build --operating-system WINDOWS_2012 --build-root .\GameLiftPackageWindows\ --name MultiplayerSample --build-version v1.0 --region us-west-2
+`
+This returns the build id which will be used in the next steps. Example Build ID: build-f447776f-b4ba-4f1e-99cf-09d49f114942
 
-1. From another terminal within the root of the MultiplayerSample project, start the server launcher (assumes the executable is the result of a `profile` build):
-    ```sh
-    ./build/windows/bin/profile/MultiplayerSample.ServerLauncher.exe --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --loadlevel=NewStarbase
-    ```
-Note: You may be inclined to move these cvars into a cfg file and start the game by passing the --console-command-file parameter, but don't. Some cvars are used during a system component Activate(), but the console-command-file is executed after all system components have been activated. (example: sv_gameLiftEnabled is used inside AWSGameLiftServerSystemComponent::Activate()).
+### Create Fleet
+After running this command it'll take about an hour for the fleet to activate. Check the status on the GameLift dashboard. 
 
-1. Create a game session with the below command:
-    ```sh
-    aws gamelift create-game-session --endpoint-url http://localhost:8080 --maximum-player-session-count 2 --fleet-id fleet-123 --game-session-id hello-mps
-    ```
-    You should observe logs in the `GameLiftLocal` terminal which indicate it handled the create-game-session request, and see the `ServerLauncher` load its level.
+`
+aws gamelift create-fleet --region us-west-2 --name GlLyTest2F --ec2-instance-type c5.large --fleet-type ON_DEMAND --build-id build-f447776f-b4ba-4f1e-99cf-09d49f114942 --runtime-configuration "GameSessionActivationTimeoutSeconds=300, MaxConcurrentGameSessionActivations=2, ServerProcesses=[{LaunchPath=C:\game\MultiplayerSample.ServerLauncher.exe, Parameters= --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --loadlevel=NewStarbase, ConcurrentExecutions=10}]" --ec2-inbound-permissions "FromPort=33435,ToPort=33435,IpRange=0.0.0.0/0,Protocol=UDP" "FromPort=33235,ToPort=33235,IpRange=0.0.0.0/0,Protocol=UDP"
+`
+
+Record the fleet-id for the next step.
 
-1. Finally, start the game launcher with the follow command:
-    ```sh
+### Create and Join Game Session
+`
+aws gamelift create-game-session --region us-west-2 --fleet-id fleet-6ae4d52f-4a6b-4591-99da-122d37922490 --name foogamesession1 --maximum-player-session-count 3 --game-properties Key=sv_map,Value=MultiplayerSample
+`
+
+`
+aws gamelift create-player-session --region us-west-2 --game-session-id arn:aws:gamelift:us-west-2::gamesession/fleet-6ae4d52f-4a6b-4591-99da-122d37922490/gsess-f2c14a63-e5fe-462f-9f8e-6be95b9dda26 --player-id Player1
+`
+
+Take the output of create-player-session and reformat the values to the custom session token like below, use this in the game immediately because it expires after 60 seconds.
+
+Launch the game client with:
```

Comment:
Don't you:
a) Want to launch client first and get to connection screen
b) then make player session

Also you are launching the client to talk to GameLift local? which seems confusing having just made a fleet on the GameLift service above?

Reply:
Good call about opening GameLauncher before creating the player session because the player session times out after 1 minute. Fixed!

_Also you are launching the client to talk to GameLift local? which seems confusing having just made a fleet on the GameLift service above?_
This doc is for GameLift not GameLift local. Should be better now?